com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -421,6 +421,9 @@ private static CommandLine parse(String[] args) {
 
         options.addOption(null, "version", false, "Prints the version number to the output");
 
+        options.addOption(null, "build-plugins", false, "Build editor and pipeline plugins only");
```

Comment:
By using the "--build-plugins", it's easy/fast to rebuild the plugins, and copy them as needed in the extension.

Reply:
Now that I think about it, I guess it should be a "--build-artefacts=engine,plugins" to make it more future proof.